lib/ReactViews/DataCatalog/DataCatalogMember.jsx

```diff
@@ -4,7 +4,7 @@ import React from "react";
 import createReactClass from "create-react-class";
 import PropTypes from "prop-types";
 import ObserveModelMixin from "../ObserveModelMixin";
-import DataCatalogItem from "./DataCatalogItem.jsx";
+import { DataCatalogItem } from "./DataCatalogItem.jsx";
```

Comment:
Not sure why you had to change this, since we're also exporting it as default. Maybe use ES exports for DataCatalogItem.jsx instead of mixing CJS and ES?